examples/privateGPT/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.10
+
+RUN python3 -m ensurepip
+RUN pip3 install --no-cache --upgrade pip setuptools
+RUN pip install python-dotenv tqdm langchain
+RUN pip install chromadb==0.3.29 --no-deps
+
+
+WORKDIR /app
+
+#RUN git clone https://github.com/go-skynet/privateGPT
+
+WORKDIR /app/privateGPT
+
+
+COPY ./privateGPT/requirements.txt .
+
+RUN pip install -r requirements.txt
+
+
+CMD ["tail", "-f", "/dev/null"]
```

Comment:
another leftover? shouldn't use the entrypoint?

Reply:
yes, they were leftovers, the container would just stay up indefinetly, one could connect to it via docker exec -it then run the python scrpts. Tought about having a simple rest-endpoint to query it, but not sure, intention was just to get it running. How would you like to have it running ?

Reply:
updated, please tell me what you think now